common/test/acceptance/tests/discussion/test_cohort_management.py

```diff
@@ -82,18 +83,40 @@ def test_cohort_description(self):
         Given I have a course with a manual cohort and an automatic cohort defined
         When I view the manual cohort in the instructor dashboard
         There is text specifying that students are only added to the cohort manually
-        And when I vew the automatic cohort in the instructor dashboard
+        And when I view the automatic cohort in the instructor dashboard
         There is text specifying that students are automatically added to the cohort
         """
         self.verify_cohort_description(
             self.manual_cohort_name,
-            'Students are added to this group only when you provide their email addresses or usernames on this page',
+            'Students are added to this cohort group only when you provide '
+            'their email addresses or usernames on this page',
         )
         self.verify_cohort_description(
             self.auto_cohort_name,
-            'Students are added to this group automatically',
+            'Students are added to this cohort group automatically',
         )
 
+    def test_no_content_groups(self):
+        """
+        Scenario: if the course has no content groups defined (user_partitions of type cohort),
+        the settings in the cohort management tab reflect this
+
+        Given I have a course with a cohort defined but no content groups
+        When I view the cohort in the instructor dashboard and select settings
+        Then the cohort group is not linked to a content group
+        And there is text stating that no content groups are defined
+        And I cannot select the radio button to enable content group association
+        And there is a link I can select to open Group settings in Studio
+        """
+        self.cohort_management_page.select_cohort(self.manual_cohort_name)
+        self.assertIsNone(self.cohort_management_page.get_cohort_associated_content_group())
+        self.assertEqual(
+            "No content groups exist. Create a content group to associate with cohort groups. Create a content group",
+            self.cohort_management_page.get_cohort_related_content_group_message()
+        )
+        # TODO: test can't select radio button
+        # TODO: test link to Studio
```

Comment:
These are the bok choy tests that need to be finished, mentioned in standup. And the link to Studio mentioned here is different-- it takes you directly to the group configurations settings page, if no content groups have been defined.